.changelog/1432.changed.txt

```diff
@@ -0,0 +1 @@
+Remove compress_encoding and all of our own compression code in favor of using the confighttp hhelper
```

Comment:
```suggestion
feat: remove compress_encoding and all of our own compression code in favor of using the confighttp helper
```

Reply:
feat: Deprecate compress_encoding and remove all of our own compression code in favor of using the confighttp helper